src/App.test.js

```diff
@@ -1,8 +1,42 @@
-import { render, screen } from '@testing-library/react';
-import App from './App';
+import React from "react";
+import Enzyme, { shallow } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import App from "./App";
+import Home from "./pages/Home";
 
-test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("when app renders", () => {
+  it("renders a header and footer component", () => {
+    //arrange
+    const renderedApp = shallow(<App />);
+    //act
+    const renderedHeader = renderedApp.find("Header");
+    const renderedFooter = renderedApp.find("Footer");
+    // console.log("rendered header: ", renderedHeader);
```

Comment:
There are a lot of console.logs left in here. Can you clean them up?

Reply:
cleaned up!